validator_test.go

```diff
@@ -580,20 +587,39 @@ func TestValidateRequiredFailing(t *testing.T) {
 		{ErrRequired, &struct {
 			C int `validate:"required"`
 		}{}},
-		{ErrRequired, &struct {
+		{ErrEmpty, &struct {
```

Comment:
I think this one is debatable. `C` is on the config, but nil. Both `ErrEmpty` and `ErrRequired` might make sense (just in case some other change modifies this error value).